lib/use-cases/RejectDropbox.js

```diff
@@ -1,3 +1,4 @@
+const log = require('../log')();
```

Comment:
We're passing in 'log' as part of the container, so no need to import it here (same with other use-cases)

Reply:
Check out Dependencies.js, the logger is injected pretty much everywhere